src/main/java/nextstep/subway/path/application/PathService.java

```diff
@@ -18,11 +20,18 @@ public PathService(final LineRepository lineRepository, final StationRepository
         this.stationRepository = stationRepository;
     }
 
-    public PathResponse findShortestPath(final Long sourceId, final Long targetId) {
+    public PathResponse findShortestPath(final Long sourceId, final Long targetId, final Integer age) {
         Station sourceStation = findStationById(sourceId);
         Station targetStation = findStationById(targetId);
         PathFinder pathFinder = PathFinder.from(lineRepository.findAll());
-        return pathFinder.getShortestPath(sourceStation, targetStation);
+
+        if (age == null) {
+            return pathFinder.getShortestPath(sourceStation, targetStation,
```

Comment:
```suggestion
            return pathFinder.getShortestPath(sourceStation, targetStation,AGE.DEFAULT);
```

`Age.of(...)`
`null`을 계속 가지고 다니면, `null` 체크를 계속해서 해야되는 이슈가 있을듯해요!

 `AGE.DEFAULT` 같은 `기본값` 를 전달하는 방향으로 해도 좋을것 같네요!

Reply:
age에 기본값을 넣기보다 null은 null로 관리하는게 맞다고 생각했었는데, 지금 다시 생각해보니 불필요한 null 체크를 계속해야되는 비용을 고려했을 때 훨씬 합리적인 선택이라 생각되네요~! 감사합니다 👍 👍 👍